quix-frontend/client/src/lib/sql-autocomplete/adapter/sql-autocomplete-adapter.ts

```diff
@@ -48,6 +55,31 @@ export class SqlAutocompleter implements IAutocompleter {
     }
   }
 
+  public async getCompletionItemsFromQueryContextColumn(queryContext: QueryContext) {
```

Comment:
why async function?

Reply:
if the function calls async functions
i should be async aswell?

Reply:
it only returns async function, it is not using await by itself, in that case using async is unnecessary, and in our case - we just not writing it, so you can remove the async prefix specifically here